components/03-modules/range/range.js

```diff
@@ -64,5 +59,4 @@
 
```

Comment:
In fact this update with `inputMax` and `inputMin` eventListener doesn't work on my local machine

Reply:
instead of `this` use `inputMax` or `inputMin` to start working this script